packages/diagram_capture/lib/diagram_capture.dart

```diff
@@ -131,6 +131,35 @@ class _DiagramWidgetController extends WidgetController implements TickerProvide
   Future<List<Duration>> handlePointerEventRecord(List<PointerEventRecord> records) async {
     return const <Duration>[];
   }
+
+  @override
+  Future<int> pumpAndSettle([
```

Comment:
@gspencergoog Not sure this if this is the best way to do this, but I cribbed it from the framework implementation.